### ISSUE TITLE
Texture interior grey rather than white

### ISSUE THREAD

**Issue by ghost** [viktorradnai/flightgear-ask21#152]
Comparing this:
![fgfs-screen-670](https://user-images.githubusercontent.com/12275960/34373239-4d1b65ec-eada-11e7-9015-128d0961d838.png)
to
https://www.alexander-schleicher.de/wp-content/uploads/2015/03/21-Cockpit-002.jpg 
(source: alexander-schleicher.de)

shows that the interior walls are actually painted grey whereas ours are almost completely white

**Comment by ghost** [comment]
![fgfs-screen-762](https://user-images.githubusercontent.com/12275960/36632856-329dd58e-198c-11e8-80ce-e03e249f70d8.png)
WIP on better interior (grey, more detailed, correct seat shell)

**Comment by viktorradnai** [comment]
Good start, but it's a bit too dark. The seat pan and stick are light grey, say #eeeeee in HTML notation, while the fuselage is perhaps #aaaaaa. In practice I haven't even noticed the difference in colour, I had to look up the photos I took :) The canopy frame is painted black on the inside.

**Comment by ghost** [comment]
Yep, I'm gonna make the interior a bit brighter again. Do you mean it's literally painted *black* or grey as the rest? IIRC at our ASK21, it's painted grey as the rest

**Comment by viktorradnai** [comment]
Yes, it is painted matte black.

**Comment by ghost** [comment]
I suppose this depends on the owner then.. each club paints it like they want. As a compromise, I'd suggest using the colors according to the image by AS? https://www.alexander-schleicher.de/wp-content/uploads/2015/03/21-Cockpit-002.jpg
